contracts/USM.sol

```diff
@@ -1,57 +1,48 @@
 pragma solidity ^0.6.7;
 
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./BufferedToken.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
-import "./oracles/IOracle.sol";
+import "./WadMath.sol";
 import "@nomiclabs/buidler/console.sol";
+import "./FUM.sol";
 
 /**
  * @title USM Stable Coin
  * @author Alex Roan (@alexroan)
- * @notice Concept by Jacob Eliosoff (@jacob-eliosoff)
+ * @notice Concept by Jacob Eliosoff (@jacob-eliosoff).
  */
-contract USM is ERC20 {
+contract USM is BufferedToken {
     using SafeMath for uint;
+    using WadMath for uint;
 
-    uint constant UNIT = 1e27; // 10**27, MakerDAO's RAY
-    uint constant MINT_FEE = UNIT / 1000; // 0.1%
-    uint constant BURN_FEE = UNIT / 200; // 0.5%
+    uint constant MINT_FEE = WAD / 1000; // 0.1%
```

Comment:
The comments should say `// 0.001 ETH` and `// 0.005 ETH` since these are not percentages anymore.